app/src/main/java/br/com/dionataferraz/vendas/AccountViewModel.kt

```diff
@@ -0,0 +1,33 @@
+package br.com.dionataferraz.vendas
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class AccountViewModel : ViewModel() {
+
+    private val error: MutableLiveData<String> = MutableLiveData()
+    val shouldShowError: LiveData<String> = error
+    private val accountModel: MutableLiveData<AccountModel> = MutableLiveData()
+    val accountLiveData: LiveData<AccountModel> = accountModel
+
+    fun createAccount(name: String?, value: String?, responsible: String?, type: String?) {
+        if (name.isNullOrBlank())
+            error.value = "The name field cannot be empty!"
+        else if (value.isNullOrBlank())
+            error.value = "The value field cannot be empty!"
+        else if (responsible.isNullOrBlank())
+            error.value = "The responsible field cannot be empty!"
+        else if (type.isNullOrBlank())
+            error.value = "The account type field must be checked!"
+        else {
+            val accountCreated = AccountModel(
+                name = name,
+                value = value,
+                responsible = responsible,
+                type = type
+            )
+            accountModel.value = accountCreated
```

Comment:
Como tu está usando o account em um lugar apenas, nós podemos fazer dessa forma
```suggestion
        account.value = Account(
            description = description,
            value = value,
            responsible = responsible,
            credit = credit,
            debit = debit
        )
```